pkg/serverless/appsec/httpsec/proxy.go

```diff
@@ -200,6 +205,7 @@ func (lp *ProxyLifecycleProcessor) spanModifier(lastReqId string, chunk *pb.Trac
 		makeContext(
 			&ctx,
 			&event.Path,
+			&event.Path,
```

Comment:
Worth checking with the API Management team if not setting the http.route tag is preferable to let the backend decide based on http.url.

Reply:
25dd050e6e1d